duckduckgo/iOS
# fully release dax onboarding and dialogs

Task/Issue URL: https://app.asana.com/0/392891325557410/1162375367527169
Tech Design URL:
CC:

**Description**:

* Removes old onboarding, tips and experiment.
* Removes EasyTips library from the project.
* Fixes text jumping in dax dialog
* Improves clarity around new install logic and removes some redundant code
* Remove some redundant code/storyboard stuff for onboarding

**Steps to test this PR**:

1. Test migrating from before any Dax Onboarding.  No dax dialogs or onboarding should be displayed after upgrading.
1. Clean install, ensure that dax onboarding is shown and dax dialogs still behave as before (but with the text jumping fix)
1. Check logo positioning works across devices



**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
